recipes/mod_fcgid.rb

```diff
@@ -29,12 +29,15 @@
     backup false
   end
 
-  directory '/var/run/httpd/mod_fcgid' do
-    owner node['apache']['user']
-    group node['apache']['group']
-    recursive true
-    only_if { node['platform_version'].to_i >= 6 }
+  if ((node['platform_family'] == 'rhel') && (node['platform_version'].to_i == 6)) ||
```

Comment:
A library helper here called something like rhel_6? would make this statement more readable.